src/methods.jl

```diff
@@ -62,6 +72,22 @@ julia> subtract_bias(frame, bias)
 subtract_bias(frame::AbstractArray, bias_frame::AbstractArray) = subtract_bias!(deepcopy(frame), bias_frame)
 
 
+"""
+    subtract_bias(::Union{FITSIO.ImageHDU, AbstractArray}, ::Union{FITSIO.ImageHDU, AbstractArray})
+    subtract_bias(::Union{FITSIO.ImageHDU, AbstractArray}, ::String; hdu_bias = 1)
+    subtract_bias(::String, ::Union{FITSIO.ImageHDU, AbstractArray}; hdu_frame = 1)
+    subtract_bias(::String, ::String; hdu_frame = 1, hdu_bias = 1)
```

Comment:
At this point we are making the documentation confusing by writing out too many methods and using awkward unions that aren't present anywhere else.

```suggestion
    subtract_bias(frame, bias_frame; [hdu=1])

Subtract the bias frame from `frame`. If either arguments are `FITSIO.ImageHDU` they will be loaded into memory. If either arguments are strings we will attempt to locate a FITS file and open it before loading the data from the given `hdu`. If loading multiple files, you can specify the HDU numbers separately (`hdu=(1, 2)`) or simultanesously (`hdu=1`).
```